docs/reference/edot-collector/config/tail-based-sampling.md

```diff
@@ -0,0 +1,93 @@
+---
+navigation_title: Tail-based Sampling
+description: Configuration of the EDOT Collector for Tail-based sampling.
+applies_to:
+stack:
+serverless:
+observability:
+product:
+edot_collector: ga
+products:
+- id: cloud-serverless
+- id: observability
+- id: edot-collector
+---
+
+# Configure tail-based sampling
+
+Tail-based sampling is where the decision to sample a trace takes place by considering all or most of the spans within the trace. This is in contrast to Head-based sampling, which is on the client-side where traces are created.
+
+Because metrics are calculated and aggregations are performed on trace data in the EDOT collector, some specific configuration is required to ensure that the tail sampling decisions are made after these calculations.
+Otherwise, the metrics and aggregations would be incorrect due to the partial representation of traces.
+
```

Comment:
```suggestion
Tail-based sampling analyzes a complete trace before deciding whether to keep it, enabling intelligent decisions based on factors like errors or high latency. This is superior to head-based sampling, which makes a premature decision at the start of a trace.

Within the OpenTelemetry Collector, any processor that generates metrics from traces must run before the tail-sampling processor. If sampling happens first, metrics will be calculated on an incomplete data set, leading to inaccurate and misleading reporting.
```

Reply:
I wouldn't use the word "superior" because it really depends on the use case. In some cases, probabilistic sampling on the client side is the right choice for the scenario.

Reply:
Agreed, edited to make it more neutral.